display.h

```diff
@@ -207,7 +207,8 @@ enum s9x_getdirtype
 	IPS_DIR,
 	BIOS_DIR,
 	LOG_DIR,
-	LAST_DIR
+	LAST_DIR,
```

Comment:
LAST_DIR should always be the last entry in the list, it is just a marker for the end of the enum.